app/api_clients/github.js

```diff
@@ -29,6 +32,31 @@ class GithubAPIClient extends GithubAPI {
 
       return Promise.resolve(user.github_access_token);
     };
+
+    async function paginate(method, params = {}) {
```

Comment:
Should the name of this function be something like `getAllPages`?

Reply:
yes it should!